src/Swashbuckle.AspNetCore.SwaggerGen/SwaggerGenerator/SwaggerGenerator.cs

```diff
@@ -650,5 +655,33 @@ private OpenApiMediaType CreateResponseMediaType(ModelMetadata modelMetadata, Sc
             new KeyValuePair<string, string>("5\\d{2}", "Server Error"),
             new KeyValuePair<string, string>("default", "Error")
         };
+
+        private string GenerateSummary(ApiDescription apiDescription)
+        {
+            string operationSummary = null;
+#if NET7_0_OR_GREATER
```

Comment:
Maybe it would be neater to `#if` the two property assignments and the entire declaration of these methods instead? The net result should be the same for .NET <7 that the two properties have a value of `null`?